.idea/jsLibraryMappings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
all the files under `.idea` are your IDE related files. They should not be included in the change. You can create a `.gitignore` file and add `.idea` to it and then git will stop tracking changes to those files.